### PR TITLE
try fixed image

### PR DESCRIPTION
**Pull request by serhiykrupka**

### CODE REVIEW

**Comment by vitech-sdlc** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **serhiy-krupka-vitech**
You can assign the PR to them by writing `/assign @serhiy-krupka-vitech` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=vitech-team%2Fmood-feed-frontend).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/vitech-team/mood-feed-frontend/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by oleg-ilechko** [comment]
I am comment!

**Comment by vitech-sdlc** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[serhiy-krupka-vitech](https://github.com/vitech-team/mood-feed-frontend/pull/13#issuecomment-689587324 "Approved")*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=vitech-team%2Fmood-feed-frontend).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com#issuecomment-689587324/vitech-team/mood-feed-frontend/blob/master/OWNERS)~~ [serhiy-krupka-vitech]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by vitech-sdlc** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[oleg-ilechko](https://github.com/vitech-team/mood-feed-frontend/pull/13#issuecomment-689588149 "Approved")*, *[serhiy-krupka-vitech](https://github.com/vitech-team/mood-feed-frontend/pull/13#issuecomment-689587324 "Approved")*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=vitech-team%2Fmood-feed-frontend).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com#issuecomment-689588149/vitech-team/mood-feed-frontend/blob/master/OWNERS)~~ [oleg-ilechko,serhiy-krupka-vitech]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by vitech-sdlc** [comment]
@serhiy-krupka-vitech: cat image

<details>

In response to [this](https://github.com/vitech-team/mood-feed-frontend/pull/13#issuecomment-689589485):

>/meowvie clothes


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository.
</details>

**Comment by vitech-sdlc** [comment]
@oleg-ilechko: [dog image](https://storage.googleapis.com/this-is-fine-images/this_is_fine.png)

<details>

In response to [this](https://github.com/vitech-team/mood-feed-frontend/pull/13#issuecomment-689589860):

>/this-is-fine


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository.
</details>

File: jenkins-x.yml
Comment (oleg-ilechko) [inline jenkins-x.yml:9]: Need work

File: jenkins-x.yml
Comment (oleg-ilechko) [inline jenkins-x.yml:14]: And here!

**Review by oleg-ilechko — CHANGES_REQUESTED** [review]
Need some work

**Comment by vitech-sdlc** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[serhiy-krupka-vitech](https://github.com/vitech-team/mood-feed-frontend/pull/13#issuecomment-689587324 "Approved")*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=vitech-team%2Fmood-feed-frontend).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/vitech-team/mood-feed-frontend/blob/master/OWNERS)~~ [serhiy-krupka-vitech]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by vitech-sdlc** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[oleg-ilechko](https://github.com/vitech-team/mood-feed-frontend/pull/13#issuecomment-689595051 "LGTM")*, *[serhiy-krupka-vitech](https://github.com/vitech-team/mood-feed-frontend/pull/13#issuecomment-689587324 "Approved")*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=vitech-team%2Fmood-feed-frontend).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com#issuecomment-689595051/vitech-team/mood-feed-frontend/blob/master/OWNERS)~~ [oleg-ilechko,serhiy-krupka-vitech]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by vitech-sdlc** [comment]
:star: PR built and available in a preview environment **vitech-team-mood-feed-frontend-pr-13** [here](http://mood-feed-frontend-jx-vitech-team-mood-feed-frontend-pr-13.35.224.13.149.nip.io)

**Pull request merged**
